test/basic.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+. ${top_srcdir}/test/test-common.sh
```

Comment:
Addressed in this file and other .sh files, in commit: e5812284788adb558f15f71ca4851cde668ddff6